doc: Add device registration troubleshooting (#582)
TheThingsIndustries/lorawan-stack-docs: doc/content/devices/troubleshooting/_index.md

<!--more-->

> ### I cannot register my end device. Why?

Reviewer: ```suggestion
### I receive an error when registering an end device
```

> Not being able to finish the registration of your end device can be due to some common errors:

Reviewer: ```suggestion
Here are some common errors and solutions:
```

> - **ID already taken**: This error occurs if you try to register an end device using a Device ID that is already used for another device in the same application. Choose a different Device ID

Reviewer: ```suggestion
- **ID already taken**: Another end device in the application is using this ID. Choose a different Device ID. See [ID and EUI constraints]({{< ref "reference/id-eui-constraints" >}}) for more information.
```

> - **An end device with JoinEUI `<join-eui>` and DevEUI `<dev-eui>` is already registered as `<device-id>` in application `<application-id>`**: Within {{% tts %}} deployment, only a single end device can be registered with a certain combination of DevEUI and JoinEUI. If you previously registered an end device with this combination of DevEUI and JoinEUI, you can delete the existing device and repeat the registration. You can also register the device using another AppEUI, but keep in mind that you will need to configure that AppEUI in the device as well

Reviewer: ```suggestion
- **An end device with JoinEUI `<join-eui>` and DevEUI `<dev-eui>` is already registered as `<device-id>` in application `<application-id>`**: Within {{% tts %}} deployment, only a single end device can be registered with a certain combination of DevEUI and JoinEUI/AppEUI. If you previously registered an end device with this combination of DevEUI and JoinEUI, you can delete the existing device and repeat the registration. You can also register the device using another JoinEUI, but keep in mind that you will need to configure that JoinEUI in the device as well.
```

> - **An end device with JoinEUI `<join-eui>` and DevEUI `<dev-eui>` is already registered in another tenant**: Within {{% tts %}} deployment, only a single end device can be registered with a certain combination of DevEUI and JoinEUI, across all tenants. If someone already registered an end device with this combination of DevEUI and JoinEUI, you can contact the manufacturer to check if your EUIs are correct and/or provide you new EUIs. You can also register the device using another AppEUI, but keep in mind that you will need to configure that AppEUI in the device as well

Reviewer: ```suggestion
- **An end device with JoinEUI `<join-eui>` and DevEUI `<dev-eui>` is already registered in another tenant**: Within {{% tts %}} deployment, only a single end device can be registered with a certain combination of DevEUI and JoinEUI, across all tenants. If someone already registered an end device with this combination of DevEUI and JoinEUI, you can contact the manufacturer to check if your EUIs are correct and/or provide you new EUIs. You can also register the device using another AppEUI, but keep in mind that you will need to configure that AppEUI in the device as well.
```
